src/components/Visualizer/Visualizer.module.scss

```diff
@@ -1,12 +1,12 @@
 .visualizer {
     position: absolute;
-    width: 90%;
-    height: 86%;
+    width: 80%;
```

Comment:
Why did you changed these css values?

Reply:
For the CSS part, I just applied a quick fix that Askat suggested on Discord. I did not check if those values where necessary or not.
I can remove all the CSS styles I made and someone else can properly fix the CSS issues in another PR.  
For the other issues:
1. Unfortunately, I think it is the only way to achieve the recording. It is recording live on what the canvas is showing. It is like taking a video of the computer screen using your smartphone. If nothing is shown, nothing can be recorded. I tried different things but I could not find any other solution. Maybe someone will find another way but it is all I got for now. 
2. Yes, there is maybe a bit of lag. The frame rate of the recording can be changed easily. I will increase the frame rate value and it should be better.
 3. I look around but I did not find any solution for that at the moment.